src/cli/publish_banner.ts

```diff
@@ -0,0 +1,18 @@
+const bold = '\u001b[1m'
+const green = '\u001b[32m'
+const cyan = '\u001b[36m'
+const underline = '\u001b[4m'
+const reset = '\u001b[0m'
+
+export default `${bold}${green}┌──────────────────────────────────────────────────────────────────────────┐${reset}
+${bold}${green}│${reset}${reset} Share your Cucumber Report with your team at ${underline}${bold}${cyan}https://reports.cucumber.io${reset} ${bold}${green}│${reset}
+${bold}${green}│${reset}                                                                          ${bold}${green}│${reset}
+${bold}${green}│${reset} Command line option:    ${cyan}--publish${reset}                                        ${bold}${green}│${reset}
+${bold}${green}│${reset} Environment variable:   ${cyan}CUCUMBER_PUBLISH_ENABLED${reset}=${cyan}true${reset}                    ${bold}${green}│${reset}
+${bold}${green}│${reset}                                                                          ${bold}${green}│${reset}
+${bold}${green}│${reset} More information at ${underline}${bold}${cyan}https://reports.cucumber.io/docs/cucumber-js${reset}         ${bold}${green}│${reset}
+${bold}${green}│${reset}                                                                          ${bold}${green}│${reset}
+${bold}${green}│${reset} To disable this message, add this to your ${bold}./cucumber.js${reset}:                 ${bold}${green}│${reset}
+${bold}${green}│${reset} ${bold}module.exports = { default: '--publish-quiet' }${reset}                          ${bold}${green}│${reset}
+${bold}${green}└──────────────────────────────────────────────────────────────────────────┘${reset}
```

Comment:
This looks tough to maintain. Let me take a look at updating this to use libraries for the ANSI fonts and the table border.

Reply:
Updated. Note removed the bold as without it, it looks a bit better on my machine (top bold, bottom non-bold). Let me know if we really want to keep the bold:

![Screen Shot 2020-09-07 at 3 55 52 PM](https://user-images.githubusercontent.com/1676758/92419512-d7355480-f122-11ea-8a6c-7686ebadf0d5.png)

Reply:
Nice thanks!